keywords.robot

```diff
@@ -605,6 +605,18 @@ Prepare To Serial Connection
         Set Global Variable    ${PLATFORM}
     END
     Get DUT To Start State
+    # the following TRY/EXCEPT exposes telnet connection error pre-emptively
+    TRY
+        Telnet.Set Timeout    1
+        Telnet.Read
+    EXCEPT    EOFError: telnet connection closed
+        ${err_msg}=    Catenate    SEPARATOR=\n
+        ...    - telnet (or minicom on RTE) connection might be opened - either by you or someone else
+        ...    - Verify and close your connections before starting tests
+        ...    - stop minicom processes on RTE, restart ser2net service (systemctl restart ser2net)
+        Fail    ${err_msg}
+    END
+    Telnet.Set Timeout    300
```

Comment:
```suggestion
        ${old_timeout}=    Telnet.Set Timeout    1
        Telnet.Read
    EXCEPT    EOFError: telnet connection closed
        ${err_msg}=    Catenate    SEPARATOR=\n
        ...    - telnet (or minicom on RTE) connection might be opened - either by you or someone else
        ...    - Verify and close your connections before starting tests
        ...    - stop minicom processes on RTE, restart ser2net service (systemctl restart ser2net)
        Fail    ${err_msg}
    END
    Telnet.Set Timeout    ${old_timeout}
```

Reply:
i removed the timeout part completely: https://github.com/Dasharo/open-source-firmware-validation/pull/382/commits/83718387641b490c0897b2ba53518261f9dd99a8